### PR TITLE
Del 202 move ews scripts

### PR DESCRIPTION
**Pull request by mikenairn**
Moves all the delorean ews scrips that were once in the delorean-ews branch of the operator into this repo.

Note: These will be removed once the work to migrate them all to Golang is complete.

**Verification:**

```
MANIFESTS_DIR=/home/mnairn/go/src/github.com/integr8ly/integreatly-operator/manifests ./scripts/delorean/process-image-manifests registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-39 3scale
MANIFESTS_DIR=/home/mnairn/go/src/github.com/integr8ly/integreatly-operator/manifests ./scripts/delorean/process-csv-images 3scale
MANIFESTS_DIR=/home/mnairn/go/src/github.com/integr8ly/integreatly-operator/manifests ./scripts/delorean/mirror-images.sh
```

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
👀

**Comment by StevenTobin** [comment]
Seeing this error

`open /home/stobin/go/src/github.com/integr8ly/integreatly-operator/manifests/integreatly-3scale/3scale-operator.package.yaml: no such file or directory`

The 3scale package is `3scale.package.yaml` but we're setting it to 

`CURRENT_CSV_FILE_NAME=${PRODUCT}-operator.package.yaml`

I'm confused now how this worked before @laurafitzgerald

File: scripts/delorean/process-image-manifests [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+extract_manifests_from_image() {
+    echo "Creating ${MANIFEST_TMP_DIR} directory"
+    mkdir -p ${MANIFEST_TMP_DIR}
+    echo "Extracting manifests from image '${IMAGE}' to ${MANIFEST_TMP_DIR}"
+    oc image extract $IMAGE --path /manifests/:${MANIFEST_TMP_DIR}
+    ls -la ${MANIFEST_TMP_DIR}
+}
+
+get_current_csv_filename() {
```
Comment (laurafitzgerald) [inline scripts/delorean/process-image-manifests]: @mikenairn I thought we removed this one. It was causing a bug. The 3scale package is 3scale.package.yaml nbot 3scale-operator.package.yaml

Comment (mikenairn) [reply scripts/delorean/process-image-manifests]: Doesn't look like i committed whatever changed i made in the delorean-ews branch :man_shrugging:

**Comment by mikenairn** [comment]
@StevenTobin That issue shoudl be fixed now. Not sure how my test worked since i tested it with 3scale, bit odd.

**Pull request merged**